### PR TITLE
Resolving the Show Description Ticket

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#89247](http://central.tri.be/issues/89247)
---

### CODE REVIEW

File: src/Tribe/Tickets.php [hunk 1]
```diff
@@ -882,9 +882,8 @@ public function ajax_refresh_settings() {
 			$ticket->description      = isset( $data['ticket_description'] ) ? esc_html( $data['ticket_description'] ) : null;
 			$ticket->price            = ! empty( $data['ticket_price'] ) ? filter_var( trim( $data['ticket_price'] ), FILTER_SANITIZE_NUMBER_FLOAT, FILTER_FLAG_ALLOW_FRACTION | FILTER_FLAG_ALLOW_THOUSAND ) : 0;
 			$ticket->purchase_limit   = isset( $data['ticket_purchase_limit'] ) ? absint( $data['ticket_purchase_limit'] ) : apply_filters( 'tribe_tickets_default_purchase_limit', 0, $ticket->ID );
-
-			$show_description = isset( $data['ticket_show_description'] ) ? absint( $data['ticket_show_description'] ) : 0;
-			$ticket->show_description = $ticket->show_description();
+			$ticket->show_description = isset( $data['ticket_show_description'] ) ? 'yes' : 'no';
```
Comment (zbtirrell) [inline src/Tribe/Tickets.php]: everywhere else this seems to use `1` or `0` (maybe `true`/`false` - why introduce `yes` and `no` here?

Comment (zbtirrell) [reply src/Tribe/Tickets.php]: why not call `$ticket->show_description();` here like in Tickets Plus?

File: src/Tribe/RSVP.php [hunk 2]
```diff
@@ -926,6 +926,10 @@ public function save_ticket( $event_id, $ticket, $raw_data = array() ) {
 			return false;
 		}
 
+		// Updates if we should show Description
+		update_post_meta( $ticket->ID, tribe( 'tickets.handler' )->key_show_description, $ticket->show_description );
```
Comment (zbtirrell) [inline src/Tribe/RSVP.php]: same concern here as in https://github.com/moderntribe/event-tickets-plus/pull/436

File: src/Tribe/Tickets.php (hunk 1 above)
Comment (bordoni) [reply src/Tribe/Tickets.php]: Because on this case it is where we initially start to save the data that is sent to ET+

Comment (bordoni) [reply src/Tribe/Tickets.php]: For consistency.  I used `yes`/`no` like WooCommerce uses.

**Pull request merged**